cards.js

```diff
@@ -16,8 +28,47 @@ function shuffle(array){
   return array;
 }
 
-function buildDeck(){
-
-// Put some code in here!
+var deck = []
+function buildDeck(suits, values){
+  for(var index_v in values){
+    for(var index_s in suits){
+      var card = {suit: suits[index_s], value: values[index_v]};
+      deck.push(card);
+    }
+  }
+  return deck
+}
```

Comment:
i love the semantic names for your indexing, so many times developers forget about semantic naming with regard to indexes, here its very clear which index is for which iteration